.github/workflows/spellcheck.yaml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run a spellcheck on the codebase.
+# It runs a few days before each release. At 00:00 on day-of-month 27 in March, June, September, and December.
+
+name: Run Spellcheck
+
+on:
+  schedule:
+    - cron: "0 0 27 3,6,9,12 *"
+
+permissions:
+  contents: read
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: development
```

Comment:
if this is the git ref: `develop` ?

can we remove the following empty line?

Reply:
> if this is the git ref: `develop` ?

Of course :facepalm: -> fixed :heavy_check_mark: 

> can we remove the following empty line?

Removed :heavy_check_mark: